kolibri/plugins/learn/assets/src/views/content-card/index.vue

```diff
@@ -1,31 +1,16 @@
 <template>
 
-  <router-link :to="link" class="card">
-
-    <div class="card-thumbnail" :style="cardThumbnail">
-      <content-icon v-if="!thumbnail" :kind="kind" :style="iconSize" class="card-thumbnail-backup"/>
-      <div v-show="progress > 0" class="card-progress-icon-wrapper">
-        <progress-icon :progress="progress"/>
-      </div>
-
-      <div class="card-content-icon-background" :class="backgroundClass"></div>
-      <div class="card-content-icon-wrapper">
-        <content-icon :kind="kind" class="card-content-icon"/>
-      </div>
-
-      <div class="card-progress-bar-wrapper">
-        <div
-          class="card-progress-bar"
-          :style="{ width: `${progress * 100}%` }"
-          :class="{ 'card-progress-bar-mastered': mastered, 'card-progress-bar-progress': inProgress }">
-        </div>
-      </div>
-    </div>
-
-    <div class="card-text">
-      <h3 class="card-title">{{ title }}</h3>
-      <h4 v-if="subtitle" class="card-subtitle">{{ subtitle }}</h4>
-    </div>
+  <router-link :to="link" class="card" :class="{ 'mobile-card': isMobile }">
+
+    <card-thumbnail
+      class="thumbnail"
```

Comment:
Instead of attaching this `thumbnail` class, you can just add ` position: absolute` to the `mobile-thumbnail` class inside card-thumbnail

Reply:
I feel like the parent should be responsible for choosing `position` - the child shouldn't really care

Reply:
Is there a case where it would not be absolute? I guess in terms of `position` your reasoning makes sense.

Reply:
I tend to think of components similar to native HTML elements - they have reasonable default CSS built-in, but can be overridden by the parent.

For example, an `<h1>` by default might have

```stylus
display: block
margin-bottom: 16px
```

and an `<a>` by default might have

```stylus
display: inline
margin: 0
```

However either of those can be overridden.

For this specific case it's true we could just make the default `position: absolute`, but it feels weird to me. Maybe for floating toolbars and pop-ups, but not something that looks like a standard block...

Reply:
Makes sense. :100: